riot/riot.py

```diff
@@ -168,6 +169,10 @@ def create_venv(self, recreate: bool, path: t.Optional[str] = None) -> str:
             )
             return venv_path
 
+        if recreate:
```

Comment:
Since we are already testing for the directory on the previous `if`, perhaps we can rewrite that as

~~~
if os.path.isdir(venv_path)
    if recreate:
        ...
    else:
        ...
~~~
and remove the `ignore_errors=True`. WDYT?

Reply:
:+1:

Reply:
:+1: